packages/api/app/auth.js

```diff
@@ -35,29 +50,24 @@ async function ensureAuthorized(req, res, next) {
     return res.boom.unauthorized('Missing token');
   }
 
-  let userName;
-  let accessToken;
-  try {
-    ({ username: userName, accessToken } = verifyJwtToken(jwtToken));
+  if (launchpadProtectedAuth()) {
+    return ensureLaunchpadAPIAuthorized(req, res, next);
+  }
 
-    const userModel = new User();
-    const access = new AccessToken();
+  try {
+    const { username, accessToken } = verifyJwtToken(jwtToken);
```

Comment:
Is there any reason to not use `verifyJwtAuthorization`? Then return the user unauthorized error if the instance of the caught error is `TokenUnauthorizedUserError`, as we do for the other error types?

Reply:
`verifyJwtAuthorization` does not return the username, which we need to set in `req.authorizedMetadata` on line 67.

Reply:
It's tempting ot refactor `verifyJwtAuthorization` but that's out of scope here